docs/application/dotnet/guides/nui/linear-layout.md

```diff
@@ -0,0 +1,100 @@
+# Linear Layout
+
+`LinearLayout` is a linear box layout in which the children of a layout are arranged vertically or horizontally. You can set the layout direction vertically or horizontally by using `LinearOrientation` property.
+
+If you want to position the children in a linear form that is one after the other, you can use `CellPadding`. `CellPadding` is used to insert a space between each child. Unlike generic padding, `CellPadding` does not insert a space at the start, end, top, or bottom of the layout.
+
+![LinearLayout](./media/linearTotal.png)
+
+`LinearAlignment` defines from where you can start positioning children. It is very useful if children do not use all the space of a parent.
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LinearOrientation`    | LinearLayout.Orientation | Gets or sets the vertical or horizontal orientation of the linear layout |
+| `LinearAlignment`      | LinearLayout.Alignment  | Gets or sets the global child alignment |
+| `CellPadding`          | Size2D      | Gets or sets the horizontal or vertical spacing between the cells |
+
+
+## Orientation
+
+`LinearOrientation` indicates the direction of a child layout such as horizontal or vertical. The default value is horizontal.
+
+| Horizontal | Vetical |
+| -----------------------| --------------- |
+| ![Horizontal](./media/horizontalLayout.png) | ![Vertical](./media/verticalLayout.png)  |
+
+```csharp
+View layoutView = new View();
+var linearLayout = new LinearLayout();
+linearLayout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+layoutView.Layout = layout;
+```
+
+
+## Alignment
+
+`LinearAlignment` handles how the layout items are positioned within their parent layout. By default, items are aligned at **Top** and **Begin**.
+You can choose from the following alignment options:
+
+| Alignment  | Description        |
+| ---------- | ------------------ |
+| Begin      | At the left or right edge of the container, according to Left to Right (LTR) or Right to Left (RTL) direction for horizontal orientation |
+| End        | At the right or left edge of the container, according to LTR or RTL direction for horizontal orientation |
+| CenterHorizontal | At the horizontal center of the container (vertical will be set to top) |
+| Top        | At the top edge of the container |
+| Bottom     | At the bottom edge of the container |
+| CenterVertical | At the vertical center of the container (horizontal will be set to begin) |
+| Center    | At both the vertical and horizontal center of the container |
+
+![LinearAlignment](./media/linearAlignment.png)
+
+```csharp
+View layoutView = new View();
+var linearLayout = new LinearLayout();
+linearLayout.LinearAlignment = LinearLayout.Alignment.Center;
+layoutView.Layout = layout;
+```
+
+
+## CellPadding
+
+`CellPadding` is to set the padding between cells in the layout. It is used to insert a space between each child.
```

Comment:
I added the sentence : 
The type of `CellPadding` is [Size2D](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Size2D.html), which is composed of two-dimensional size. After setting `CellPadding` to parent view, the interval between children is located by the width of `CellPadding` in the case of horizontal layout or by the height of `CellPadding` in the case of vertical layout.

+ The type of CellPadding is not 'Extents' which has start, end, top, and bottom, but 'Size2D' (float width, float height).
And in the image below, the arrow is width (10) because it's horizontal layout.

Reply:
I added two sentences more. Could you check it again?
I think they should be reviewed more. :)...